### ISSUE TITLE
Issue with "WAI-ARIA Roles": incomplete section

### ISSUE THREAD

**Issue by mfrancis33** [mdn/content#4776]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles

#### What information was incorrect, unhelpful, or incomplete?
A section's content is incomplete, stopping mid-sentence.

#### Specific section or headline?
> ARIA: tabpanel role

#### What did you expect to see?
I expected to see an explanation on the `tabpanel` role of ARIA roles, but the content was missing.  
Here is what is currently there:
> The ARIA tabpanel role indicates

#### Did you test this? If so, how?
No, I did not test the static HTML text.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/aria/roles`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/roles/index.html
* Last commit: https://github.com/mdn/content/commit/1e8c048d86584064ca418a214063cb965fab3e71
* Document last modified: 2021-03-23T09:11:33.000Z

</details>

**Comment by estelle** [comment]
Complete! 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/
